olive/hardware/accelerator.py

```diff
@@ -263,10 +267,15 @@ def normalize_accelerators(system_config: "SystemConfig", skip_supported_eps_che
             system_config.type in (SystemType.Local, SystemType.PythonEnvironment, SystemType.IsolatedORT)
             and not skip_supported_eps_check
         ):
-            # don't need to check the supported execution providers if there is no evaluation
-            # target is only used for evaluation
+            # skip_supported_eps_check is True here
+            # target is used so we need to check that the system supported eps are compatible with the accelerators
```

Comment:
for qnn scenario, better to mention that the local system is cpu ORT but we specify qnn ep like we discussed offline. 

From the current change, I have no idea that the local system use cpu ort.